<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenViewer</class>
 <widget class="QMainWindow" name="ScreenViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QScrollArea" name="panel">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>788</width>
         <height>546</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>22</height>
    </size>
   </property>
   <widget class="QMenu" name="menuShortcuts">
    <property name="title">
     <string>Shortcuts</string>
    </property>
    <addaction name="actionCtrlShiftEsc"/>
    <addaction name="actionPrintScreen"/>
   </widget>
   <widget class="QMenu" name="menuDisplayMode">
    <property name="title">
     <string>Display Mode</string>
    </property>
    <addaction name="action1_1"/>
    <addaction name="actionAutoScale"/>
   </widget>
   <addaction name="menuShortcuts"/>
   <addaction name="menuDisplayMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar::item{border: 0px}</string>
   </property>
  </widget>
  <action name="actionCtrlShiftEsc">
   <property name="text">
    <string>Ctrl+Shift+Esc</string>
   </property>
  </action>
  <action name="action1_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1:1</string>
   </property>
  </action>
  <action name="actionAutoScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto scale</string>
   </property>
  </action>
  <action name="actionPrintScreen">
   <property name="text">
    <string>Print Screen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
